debian/rules

```diff
@@ -36,7 +36,8 @@ else
 endif
 
 override_dh_auto_build:
-	dh_auto_build -- lp_all
+	echo "No need to build DKMS modules at packages build time."
+	echo "DKMS modules are built at packages installation time."
```

Comment:
This was actually done on purpose so launchpad tests, if the kernel modules can be built. But now we have CI so it's not really important anymore.